spec/resource.spec.js

```diff
@@ -42,8 +42,29 @@ describe('Resource', function () {
       this.resource = new Resource(User)
     })
 
-    it('returns given count', async function () {
-      expect(await this.resource.count()).to.equal(this.count)
+    it('returns given count without filters', async function () {
+      expect(await this.resource.count({})).to.equal(this.count)
+    })
+
+    it('returns given count for given filters', async function () {
+      expect(await this.resource.count({ email: 'some-not-existing-email' })).to.equal(0)
+    })
+  })
+
+  describe('#convertedFilters', function () {
+    beforeEach(function () {
+      this.resource = new Resource(User)
+    })
+
+    it('returns empty object if no filters', async function () {
+      const filters = {}
+      expect(await Resource.convertedFilters(filters)).to.deep.equal({})
+    })
+
+    it('returns converted filters, if provided', async function () {
```

Comment:
please write a test which have `+` char in the query. And test if it was escaped. This is very important.